src/instruments/src/RTPI/index.tsx

```diff
@@ -5,7 +5,7 @@ import { useSimVar } from '../Common/simVars';
 
 const RTPIDisplay = () => {
     const [ltsTest] = useSimVar('L:A32NX_OVHD_INTLT_ANN', 'Bool', 400);
-    const [rudderTrim] = useSimVar('RUDDER TRIM', 'degrees', 100);
+    const [rudderTrim] = useSimVar('L:A32NX_HYD_RUDDER_TRIM_FEEDBACK_ANGLE', 'degrees', 100);
```

Comment:
Should use FAC rudder trim position. I'm not 100% what happens when both FAC rudder trim positions are not available, but the rudder trim display will probably be blanked